event hubs migration guide.md

```diff
@@ -0,0 +1,118 @@
+# Migration Guide (EventHubs v2 to v5)
+
+## Class changes
```

Comment:
It's not incorrect to call me verbose, but I think a short summary of why there is a new library and why I would want to migrate would help establish context and set expectations.

Reply:
I added some intro text - mostly just to make sure they understood if this was the document for them (EH v2 users that just need some quick snippets for migration) or if they're new to the library (then I point them to the main package readme.md)